backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramEventFlux.java

```diff
@@ -35,7 +35,7 @@
 
     private final FluxSink<IPayload> sink;
 
-    private final Diagram initialDiagram;
+    private Diagram initialDiagram;
```

Comment:
Maybe `initialDiagram` is no longer a good name?
`currenDiagram`? `latestDiagram` (the fact that the commit message uses this term should be a hint) ?